plugins/amazonq/codetransform/jetbrains-community/tst/software/aws/toolkits/jetbrains/services/codemodernizer/utils/CodeTransformProjectUtilsTest.kt

```diff
@@ -0,0 +1,69 @@
+// Copyright 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.codemodernizer.utils
+
+import com.intellij.openapi.projectRoots.JavaSdkVersion
+import com.intellij.openapi.projectRoots.Sdk
+import com.intellij.openapi.roots.LanguageLevelProjectExtension
+import com.intellij.openapi.roots.ProjectRootManager
+import com.intellij.pom.java.LanguageLevel
+import com.intellij.testFramework.IdeaTestUtil.getMockJdk21
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.mockito.Mockito
+import software.aws.toolkits.jetbrains.services.codemodernizer.CodeWhispererCodeModernizerTestBase
+import kotlin.test.Test
+
+class CodeTransformProjectUtilsTest : CodeWhispererCodeModernizerTestBase() {
+    lateinit var projectRootManagerMock: ProjectRootManager
+    lateinit var languageLevelProjectExtensionMock: LanguageLevelProjectExtension
+    lateinit var sdkMock: Sdk
+
+    @Before
+    override fun setup() {
+        super.setup()
+        Mockito.mockStatic(LanguageLevelProjectExtension::class.java)
+
+        sdkMock = getMockJdk21()
+        languageLevelProjectExtensionMock = Mockito.mock(LanguageLevelProjectExtension::class.java)
+
+        Mockito.`when`(LanguageLevelProjectExtension.getInstance(project)).thenReturn(languageLevelProjectExtensionMock)
```

Comment:
future prefer `whenever` from mockito-kotlin

Reply:
Trying to update all instances right now to use the `import org.mockito.kotlin` important for `whenever` , `mock` and `doReturn`.

Reply:
Looks like all but `mockStatic` exist in `mockito-kotlin`. Replaced all instances and have tests passing again.